config/xopen.conf

```diff
@@ -205,6 +205,7 @@ dpc-in-data:			xml
 
 alter-statement:			warning		# should not be used ...
 comment-paragraphs:			warning		# should not be used ...
+control-division:			unconformable	# not verified yet
```

Comment:
definitely unconformable; if possible have a look at the dialect's documentation if new entries are supported somewhere; it _think_ in this case it isn't.